tools/ZydisFuzzShared.c

```diff
@@ -136,6 +136,14 @@ void ZydisPrintInstruction(const ZydisDecodedInstruction* instruction,
 
 #endif
 
+void ZydisValidateImmediateSize(ZyanU64 value) {
+    if (value != 0 && value != 8 && value != 16 && value != 32 && value != 64) {
```

Comment:
```suggestion
void ZydisValidateImmediateSize(ZyanU64 value)
{
    if ((value != 0) && (value != 8) && (value != 16) && (value != 32) && (value != 64))
    {
```

Looks good now! Just another small suggestion regarding code style to be consistent with the rest of the code.